app/views/layouts/application.html.erb

```diff
@@ -44,7 +44,7 @@
     <%= render"shared/header" %>
     <div class="container">
       <% if flash[:alert] || flash[:error] %>
-          <div class="alert alert-error">  
```

Comment:
Removing this type of whitespace is popping up in a lot of reviews - if you ever want to just remove it from all files, feel free to send me a pull request that does that, so these differences don't creep into every review